autopush/settings.py

```diff
@@ -136,15 +144,21 @@ def __init__(self,
         self.router_table = get_router_table(router_tablename,
                                              router_read_throughput,
                                              router_write_throughput)
-        self.storage_table = get_storage_table(storage_tablename,
-                                               storage_read_throughput,
-                                               storage_write_throughput)
-        self.message_table = get_message_table(message_tablename,
-                                               message_read_throughput,
-                                               message_write_throughput)
+        self.storage_table = get_storage_table(
+            storage_tablename,
+            storage_read_throughput,
+            storage_write_throughput)
+        self.message_table = get_rotating_message_table(
```

Comment:
Nit: Might want to note that the message table needs to be created before running, otherwise badness. I'm presuming that there's a creator eventually.